tests/Integration/Metadata/DateArchiveTest.php

```diff
@@ -187,11 +187,4 @@ public function tear_down(): void {
 		update_option( 'date_format', 'F j, Y' ); // reset to default.
 		update_option( 'time_format', 'g:i a' ); // reset to default.
 	}
-
-	/**
-	 * Runs once after all tests.
-	 */
-	public static function tear_down_after_class(): void {
```

Comment:
Can you explain why this was removed? It wasn't useful?

Reply:
As per typing on line 25 we can't assign `null` ..... removing it was easier then changing the types and then add checks in rest of the code.